index.html

```diff
@@ -322,21 +322,12 @@
           for <a>pointerlockerror</a> events.</p>
         </dd>
 
-        <dt>readonly attribute Element? pointerLockElement</dt>
-        <dd>
-          <dfn title="pointerLockElement"></dfn>
-
-          <p>Returns the element set as the <a>target</a> for mouse events
-          while the pointer is locked. Null if lock is pending, pointer is
-          unlocked, or if the target is in another document.</p>
-        </dd>
-
         <dt>void exitPointerLock ()</dt>
         <dd>
           <dfn title="exitPointerLock"></dfn>
 
           <p>Initiates an exit from pointer lock state if currently locked
-          to a target in this document,
+          to a target in this shadow-including document,
```

Comment:
I recommend to define "A and B are *in the same shadow-including document* if A and B are in a shadow-including document, and A and B share the same shadow-including root"